HCK04_2018_Seattle/Projects/README.md

```diff
@@ -1,8 +1,20 @@
 # How to create a new project
 
-- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea
-- Post questions about the project idea and team on the [Hackathon forum][forum].
-- When you are ready, add a new entry in the list of **Projects** by creating a new `README.md` file in subfolder in `Projects` folder, and copying contents of [project description template][project-description-template] file into it. Step-by-step instructions for this are:
+## First steps:
+
+- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea.
+- Post questions about project ideas and teams on the [Hackathon forum][forum].
+
+## Designing your team and project:
+
+To facilitate the discussions around project ideas, you can ask the [Hackathon forum maintainers][maintainers]  to create a sub-team for you.  The subteams that already exist are listed [here][subteams].
+- Use the main [Hackathon forum][forum] to discuss project scope and team membership.
+- As you work, distill your ideas into a [pinned discussion][pinned-discussion].  If you use the [project description template][project-description-template], you'll be all set up for the next step (below).
+- Ultimately, this space is meant to serve your team, not become a burden, so you should use it however you want.
```

Comment:
@nicain  I think this one should be good one:
```
- Use the main [Hackathon forum][forum] (or your sub-team forum) to discuss project scope and team membership.
```

what do you think ?

Reply:
Ill make the change.

Reply:
OK, sub-team change made. Ill wait on you to merge or request more changes.